### ISSUE TITLE
Add a database backup

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#5]
### Who

- [x] Anonymous visitor
- [x] Doctor
- [x] Content editor
- [x] Marketer
- [x] Administrator

### Needs what

To run the project with a database.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #4

### Blockers

This issue is blocked by the following issue(s): #`issue number`
